yt/visualization/base_plot_types.py

```diff
@@ -217,34 +217,43 @@ def __init__(self, fsize, axrect, caxrect, zlim, figure, axes, cax):
 
     def _init_image(self, data, cbnorm, cblinthresh, cmap, extent, aspect):
         """Store output of imshow in image variable"""
+        cbnorm_kwargs = dict(
+            vmin=float(self.zmin) if self.zmin is not None else None,
+            vmax=float(self.zmax) if self.zmax is not None else None,
+        )
         if cbnorm == "log10":
-            norm = matplotlib.colors.LogNorm()
+            cbnorm_cls = matplotlib.colors.LogNorm
         elif cbnorm == "linear":
-            norm = matplotlib.colors.Normalize()
+            cbnorm_cls = matplotlib.colors.Normalize
         elif cbnorm == "symlog":
             if cblinthresh is None:
                 cblinthresh = float((np.nanmax(data) - np.nanmin(data)) / 10.0)
 
-            symlog_kwargs = dict(
-                linthresh=cblinthresh,
-                vmin=float(np.nanmin(data)),
-                vmax=float(np.nanmax(data)),
+            cbnorm_kwargs.update(
+                dict(
+                    linthresh=cblinthresh,
+                    vmin=float(np.nanmin(data)),
+                    vmax=float(np.nanmax(data)),
+                )
             )
             MPL_VERSION = LooseVersion(matplotlib.__version__)
             if MPL_VERSION >= "3.2.0":
                 # note that this creates an inconsistency between mpl versions
                 # since the default value previous to mpl 3.4.0 is np.e
                 # but it is only exposed since 3.2.0
-                symlog_kwargs.update(dict(base=10))
+                cbnorm_kwargs.update(dict(base=10))
```

Comment:
This could be reworded as
```suggestion
                cbnorm_kwargs["base"] = 10
```

Reply:
I agree this is more clear than the original code, thanks.